kolibri/core/auth/tasks.py

```diff
@@ -792,13 +792,23 @@ def validate(self, data):
                     "roles": ", ".join(roles),
                 }
             )
+
         user_id = user_info["id"]
 
         validate_and_create_sync_credentials(
             baseurl, facility_id, username, password, user_id=user_id
         )
+        job_data["extra_metadata"]["user_id"] = user_id
+        job_data["extra_metadata"]["username"] = user_info["username"]
+
         job_data["kwargs"]["user"] = user_id
 
+        job_data["kwargs"].update(
+            dict(
```

Comment:
I believe you can drop the `dict` wrapping here. `update` should support either a dictionary or kwargs directly

Reply:
Looks like other places in the same file use `dict` so just food for thought, no need to change

Reply:
This is where, with Python, I don't know *for sure* what type(s) dict.update accepts so I used your logic and went with what I saw that ostensibly worked in others' code :laughing:

Reply:
> Looks like other places in the same file use dict so just food for thought, no need to change

I think this was my fault, I had also somehow failed to realize that you could pass **kwargs directly to `update` rather than passing in a dict.